docs/examples/driver_examples/ni_nidcpower_example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
```

Comment:
could you please make this script into an example notebook and a test that uses pyvisa-sim? (the latter is optional)